airbyte-webapp/src/packages/cloud/locales/en.json

```diff
@@ -153,6 +153,8 @@
   "credits.emailVerificationRequired": "You need to verify your email address before you can buy credits.",
   "credits.emailVerification.resendConfirmation": "We send you a new verification link.",
   "credits.emailVerification.resend": "Send verification link again",
+  "credits.lowBalance": "Your credit balance is low. You need to add more credits to prevent your syncs from being stopped",
+  "credits.zeroBalance": "All your syncs have been stopped because your credit balance is 0. Add more credits to enable your syncs again",
```

Comment:
```suggestion
  "credits.zeroBalance": "All your syncs have been stopped because your credit balance is 0. Add more credits to enable your syncs again.",
```